main.js

```diff
@@ -7,21 +7,36 @@ var inputTitle = document.querySelector('#title');
 var inputBody = document.querySelector('#body');
 var cardGrid = document.querySelector('.card-section-grid');
 
+
 //EVENT LISTENERS
 saveButton.addEventListener('click', processInput);
+window.addEventListener('keypress', checkInput);
 
 //DATA FUNCTIONS
 function processInput() {
-  var newIdea = new Idea(inputTitle.value, inputBody.value);
-  list.push(newIdea);
-  displayCard();
+  checkInput();
```

Comment:
calling checkInput here allows us to be able to FINALLY disable the button correctly! 😎